holoviews/operation/datashader.py

```diff
@@ -502,27 +519,41 @@ class trimesh_rasterize(aggregate):
                                          objects=['bilinear', None], doc="""
         The interpolation method to apply during rasterization.""")
 
+    def _precompute(self, element):
+        from datashader.utils import mesh
```

Comment:
Why the inline import?

Reply:
Requires a very specific version of datashader. Could move it to the top in a try/except, up to you.

Reply:
Ok. That is fine I think.